dotcom-rendering/src/components/MiniProfiles.stories.tsx

```diff
@@ -204,3 +205,16 @@ export const WithSeparatorLine = {
 	},
 	decorators: [centreColumnDecorator],
 } satisfies Story;
+
+export const Sectioned = {
```

Comment:
Might just be me, but `sectioned` as a term is unclear to me. Does it mean it's in a section?

Reply:
A `sectioned` mini profiles article is one which contains sections (i.e. headers which group together profiles with something in common)

I don't love the term either. I borrowed it from the Timeline element, see:

https://github.com/guardian/dotcom-rendering/blob/638b15c87e91c8d31dc8803d55e6b2de7caecccf/dotcom-rendering/src/components/Timeline.tsx#L151

and thought it was best to use consistent terminology between formats